src/javascript/JContent/ContentRoute/ContentLayout/ContentTable/reactTable/components/cells/Cells.scss

```diff
@@ -1,29 +1,47 @@
 .cellActions {
-    display: none;
+    display: flex;
     justify-content: flex-end;
     align-items: center;
     width: inherit;
     height: 48px;
 
     background-color: transparent;
+
+    visibility: hidden;
 }
 
 .cellType {
-    display: flex;
-    flex-direction: row;
+    p {
+        overflow: hidden;
+
+        text-overflow: ellipsis;
+    }
+}
```

Comment:
Just to let you know, normally the component `Typography` can manage the truncation by using `<Typography isNowrap>`. I know will have to improve the moonstone documentation to easily find this kind of behavior.

Reply:
Thank you! Added another commit to get rid of the styling and use the props instead. And thank you for the reminder should check the docs to see what's available as well.